test/integration/controller/job/job_controller_test.go

```diff
@@ -998,6 +998,123 @@ var _ = ginkgo.Describe("Job controller interacting with scheduler", ginkgo.Orde
 		})
 	})
 
+	ginkgo.When("The workload is deleted wile admitted", func() {
+		ginkgo.It("Should restore the original node selectors", func() {
+			localQueue := testing.MakeLocalQueue("local-queue", ns.Name).ClusterQueue(prodClusterQ.Name).Obj()
+			job := testingjob.MakeJob(jobName, ns.Name).Queue(localQueue.Name).Request(corev1.ResourceCPU, "2").Obj()
```

Comment:
```suggestion
			job := testingjob.MakeJob(jobName, ns.Name).Queue(localQueue.Name).Request(corev1.ResourceCPU, "2").Suspend(false).Obj()
```

If we verify the Job is suspended in L1012-L1017, should we create the Job with `susupend=false`?

https://github.com/kubernetes-sigs/kueue/blob/2a74cb034e3e3adb6403444d6566d3d8ab2acf36/pkg/util/testingjobs/job/wrappers.go#L35-L44

Reply:
sure